terraform/workspace-variables/dev.tfvars

```diff
@@ -11,4 +11,4 @@ paas_app_stopped = false
 paas_web_app_deployment_strategy = "blue-green-v2"
 paas_web_app_instances = 1
 paas_web_app_memory = 512
-paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && rails db:safe_reset && /app/bin/delayed_job start && rails s"
+paas_web_app_start_command = "bundle exec rake cf:on_first_instance db:migrate && /app/bin/delayed_job start && rails db:safe_reset && rails cron:schedule && rails s"
```

Comment:
It took me way too long to figure out that the previous version, 
```
"bundle exec rake cf:on_first_instance db:migrate cron:schedule && rails db:safe_reset && /app/bin/delayed_job start && rails s"
```
was failing because we were doing a safe reset after we schedule our jobs, which are in our db 😂 

I thought about not deleting the jobs in safe reset, but decided against it - after all, they could refer to other objects that were deleted in the soft reset.